src/firestore-tests/scholarships.rules.test.jsx

```diff
@@ -0,0 +1,55 @@
+import {
+  clearFirestoreData,
+  initializeTestApp,
+  assertFails,
+  assertSucceeds,
+  loadFirestoreRules,
+} from '@firebase/rules-unit-testing';
+import fs from 'fs';
+
+const MY_PROJECT_ID = 'scholarships-rules-test';
+const MY_AUTH = { uid: 'alice', email: 'alice@test.com' };
+const newScholarship = {
+  name: 'test-scholarship',
+  deadline: new Date(),
+  description: 'this is a test scholarship',
+  amount: {
+    type: 'UNKNOWN',
+    min: 0,
+    max: 0,
+  },
+};
+
+function getFireStore(auth = null) {
+  return initializeTestApp({ projectId: MY_PROJECT_ID, auth }).firestore();
+}
+loadFirestoreRules({
```

Comment:
A few things
- There's no JSX in this file so I'd change it to `.js`
- I would rename this file as `firestore.rules.test.js` instead so it can serve as the tests for all the firestore rules instead of just the ones related to the scholarships collection.
- `src/firestore-tests` is kind of an awkward location considering where `firestore.rules` is. I don't think it can go outside of `src/` though for `yarn test` to work so I don't have much of a better suggestion

Reply:
for now I just took the file out of the `firestore-tests` folder and put it as its own file.